.github/docker/docker-compose.yml

```
@@ -42,7 +42,7 @@ services:
     profiles: ["squid-basic-auth"]
 
   mediawiki:
-    image: registry-docker.centreon.com/docker-global/mon-mediawiki-22.10:latest
+    image: registry-docker.centreon.com/docker-global/mon-mediawiki-23.04:latest
```

this image did not change and 23.04 does not exist
```suggestion
    image: registry-docker.centreon.com/docker-global/mon-mediawiki-22.10:latest
```